### ISSUE TITLE
XLSX dependency deprecated?

### ISSUE THREAD

**Issue by kevin-arian** [DataGarage/node-xlsx-json#26]
Hi, when I try to npm install today, I hit this issue:
```
npm ERR! notarget No compatible version found: xlsx@0.7.6-g
npm ERR! notarget Valid install targets:
npm ERR! notarget 0.10.3, 0.10.2, 0.10.1, 0.10.0, 0.9.13, 0.9.12, 0.9.11, 0.9.10, 0.9.9, 0.9.8, 0.9.7, 0.9.6, 0.9.5, 0.9.4, 0.9.3, 0.9.2, 0.9.1, 0.9.0, 0.8.8, 0.8.7, 0.8.6, 0.8.5,
0.8.4, 0.8.3, 0.8.2, 0.8.1, 0.8.0, 0.7.12, 0.7.0, 0.6.1, 0.6.0, 0.5.17, 0.5.8, 0.5.7, 0.5.6, 0.5.0, 0.4.3, 0.3.10, 0.3.9, 0.3.7, 0.3.6, 0.3.5, 0.3.3
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'xlsx-to-json-lc'
```

It looks like the xlsx@0.7.6-g version is deprecated (https://libraries.io/npm/xlsx/versions?page=2).
Can this be upgraded to the latest version?

**Comment by dunklesToast** [comment]
I have fixed this here: https://www.npmjs.com/package/xlsx-to-json-depfix

**Comment by vinayaksakhare** [comment]
It is working now..Thanks

**Comment by OriginAppPriyanka** [comment]
Im facing this issue now also.. i have  
node v6.9.0
npm 3.10.8
what could be the issue? thnks

**Comment by dunklesToast** [comment]
@OriginAppPriyanka The problem is that the package.json contains a package which is deprecated. Just try it with my fixed version: 
`npm i xlsx--to-json-depfix`

**Comment by Chris2011** [comment]
why not updating the package.json to a similar or later version?

**Comment by dunklesToast** [comment]
@Chris2011 I did this. Try to install the dependency fix:
`npm i xlsx--to-json-depfix`

**Comment by Chris2011** [comment]
but it is a whole new package: https://www.npmjs.com/package/xlsx-to-json-depfix and https://www.npmjs.com/package/xlsx-to-json are different packages, why? Why not updating this package here and bring out a new version 0.2.5.? That doesn't make sense.

**Comment by Chris2011** [comment]
new version of the xlsx-to-json package not a whole new package with a fix.

**Comment by dunklesToast** [comment]
@Chris2011 I think it is because this package is no longer maintained. I mean there are 7 open Pull Requests and some are over 2 years old

**Comment by Chris2011** [comment]
Sry, I thought you were the owner of this plugin. :)

**Comment by dunklesToast** [comment]
@Chris2011 No. unfortunately not. But since I am using this package my self, I will maintain my dependency fix and merge the pull requests to it soon!
